monitoring/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 DB Netz AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#ARG BASE_IMAGE=prom/prometheus
+#FROM ${BASE_IMAGE}
+FROM debian:bullseye
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    curl
+
+FROM prom/prometheus
+
+WORKDIR /prometheus
+
+
+# RUN apk add --update curl bash
```

Comment:
Do we really need this? As it just installs `curl` on top of the Dockerfile, you can keep it as local development Dockerfile, but I would not push it to remote.